GSA/digitalgov.gov
# Refactor card-quote

## Summary

Refactored the `card-quote.html` short code to include the blockquote component classes and align its structure more closely with the `note.html` style.

## Preview

[Link to Preview]()



## Solution

The solution involved refactoring the `card-quote.html` short code to improve its structure and ensure the `blockquote` element has appropriate component classes.

- Extracted `$bg`, `$cite`, and `$text` as variables at the beginning.
- Added `card-quote__blockquote` class to the `blockquote` element.
- Ensured the logic and structure align more closely with the `note.html` style.




## How To Test
The changes made were solely refactoring. If properly implemented, there should no visual change. 
| Proposed change                                                                                  | Production                                                                                       |
| ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------ |
| [Proposed change](http://localhost:1313/2024/07/08/a-small-teams-journey-through-digital-maturity/) | [Production](https://digital.gov//2024/07/08/a-small-teams-journey-through-digital-maturity/) |








[reg-testing.md](https://github.com/user-attachments/files/16533374/reg-testing.md)
